app/models/review_reminder.rb

```diff
@@ -6,6 +6,8 @@ class ReviewReminder < ApplicationRecord
   validates :email_address, format: { with: URI::MailTo::EMAIL_REGEXP, if: -> { email_address.present? } }
   validate :review_date_cannot_be_in_the_past, if: -> { review_at.present? }
 
+  before_update :reset_reminder_sent_at_if_review_at_is_updated
```

Comment:
Code style opinion – I think this could be simplified as:

```suggestion
before_update :reset_reminder_sent_at, if: review_at_changed?
```

Reply:
yea much nicer thanks